packages/web3-core/src/web3_config.ts

```diff
@@ -22,6 +22,8 @@ export abstract class Web3Config
 		transactionBlockTimeout: 50,
 		transactionConfirmationBlocks: 24,
 		transactionPollingTimeout: 750,
+		transactionReceiptPollingInterval: 1000,
```

Comment:
I don't understand how the polling interval can be greater than the timeout, this would result in a timeout every time the polling strategy was taken